src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -110,10 +110,11 @@ def load_command_table(self, command_table):
                             raise
                 param.completer = arg.completer
 
-            command_parser.set_defaults(func=metadata.handler,
-                                        command=command_name,
-                                        _validators=argument_validators,
-                                        _parser=command_parser)
+            command_parser.set_defaults(
+                func=metadata,
+                command=command_name,
+                _validators=argument_validators,
+                _parser=command_parser)
```

Comment:
Most of our existing code base following the former pattern. Is there a reason for this change?